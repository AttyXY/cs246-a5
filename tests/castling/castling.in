game human human
move e2 e4
move e7 e5
move e1 g1
move f1 c4
move e8 c8
move d8 g5
move e1 g1
move g1 f3
move d7 d5
move e1 g1
move e8 c8
move c8 g4
move a2 a3
move b8 c6
move a3 a4
move e8 c8
quit
quit
