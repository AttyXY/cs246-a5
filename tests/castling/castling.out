WELCOME TO CHESS! 
PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]
- quit (to exit)

STARTING GAME!

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  --------
3  --------
2  PPPPPPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  ----P---
3  --------
2  PPPP-PPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppp-ppp
6  --------
5  ----p---
4  ----P---
3  --------
2  PPPP-PPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppp-ppp
6  --------
5  ----p---
4  ----P---
3  --------
2  PPPP-PPP
1  RNBQKBNR
   abcdefgh

Invalid move. Input another move.
8  rnbqkbnr
7  pppp-ppp
6  --------
5  ----p---
4  --B-P---
3  --------
2  PPPP-PPP
1  RNBQK-NR
   abcdefgh

8  rnbqkbnr
7  pppp-ppp
6  --------
5  ----p---
4  --B-P---
3  --------
2  PPPP-PPP
1  RNBQK-NR
   abcdefgh

Invalid move. Input another move.
8  rnb-kbnr
7  pppp-ppp
6  --------
5  ----p-q-
4  --B-P---
3  --------
2  PPPP-PPP
1  RNBQK-NR
   abcdefgh

8  rnb-kbnr
7  pppp-ppp
6  --------
5  ----p-q-
4  --B-P---
3  --------
2  PPPP-PPP
1  RNBQK-NR
   abcdefgh

Invalid move. Input another move.
8  rnb-kbnr
7  pppp-ppp
6  --------
5  ----p-q-
4  --B-P---
3  -----N--
2  PPPP-PPP
1  RNBQK--R
   abcdefgh

8  rnb-kbnr
7  ppp--ppp
6  --------
5  ---pp-q-
4  --B-P---
3  -----N--
2  PPPP-PPP
1  RNBQK--R
   abcdefgh

8  rnb-kbnr
7  ppp--ppp
6  --------
5  ---pp-q-
4  --B-P---
3  -----N--
2  PPPP-PPP
1  RNBQ-RK-
   abcdefgh

8  rnb-kbnr
7  ppp--ppp
6  --------
5  ---pp-q-
4  --B-P---
3  -----N--
2  PPPP-PPP
1  RNBQ-RK-
   abcdefgh

Invalid move. Input another move.
8  rn--kbnr
7  ppp--ppp
6  --------
5  ---pp-q-
4  --B-P-b-
3  -----N--
2  PPPP-PPP
1  RNBQ-RK-
   abcdefgh

8  rn--kbnr
7  ppp--ppp
6  --------
5  ---pp-q-
4  --B-P-b-
3  P----N--
2  -PPP-PPP
1  RNBQ-RK-
   abcdefgh

8  r---kbnr
7  ppp--ppp
6  --n-----
5  ---pp-q-
4  --B-P-b-
3  P----N--
2  -PPP-PPP
1  RNBQ-RK-
   abcdefgh

8  r---kbnr
7  ppp--ppp
6  --n-----
5  ---pp-q-
4  P-B-P-b-
3  -----N--
2  -PPP-PPP
1  RNBQ-RK-
   abcdefgh

8  --kr-bnr
7  ppp--ppp
6  --n-----
5  ---pp-q-
4  P-B-P-b-
3  -----N--
2  -PPP-PPP
1  RNBQ-RK-
   abcdefgh


PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO CONTINUE:
- setup (to create new setup)
- game (to continue with previous setup)
- quit (to exit)

THANKS FOR PLAYING CHESS!
Final Score:
White: 0
Black: 0
